contracts/RewardPool/modules/APR.sol

```diff
@@ -41,7 +43,10 @@ contract APR is Initializable, AccessControl {
     }
 
     function setRSI(uint256 newRSI) public onlyRole(MANAGER_ROLE) {
-        require(newRSI <= getMaxRSI(), "TOO_HIGH_RSI");
+        if (newRSI < INITIAL_RSI_BONUS) revert TooLowRSI(newRSI, INITIAL_RSI_BONUS);
```

Comment:
Actually, there is no need to have an initial RSI bonus. It can stay at zero, meaning there is no RSI bonus currently. Also you can just set it to zero when the passed value to the setter is less than 10000.